src/core/surface/version.c

```diff
@@ -36,4 +36,6 @@
 
 #include <grpc/grpc.h>
 
-const char *grpc_version_string(void) { return "0.10.1.0"; }
+const char *grpc_version_string(void) {
+	return "0.10.1.0";
+}
```

Comment:
I do not :)

It's an auto-generated file, and the code generation generates that - so
not doing that will cause a build error.

On Wed, Aug 19, 2015 at 9:36 AM Vijay Pai <notifications@github.com> wrote:

> In src/core/surface/version.c
> <https://github.com/grpc/grpc/pull/2977#discussion_r37436288>:
>
> > @@ -36,4 +36,6 @@
> >
> >  #include <grpc/grpc.h>
> >
> > -const char *grpc_version_string(void) { return "0.10.1.0"; }
> > +const char *grpc_version_string(void) {
> > +	return "0.10.1.0";
> > +}
>
> I think you want to undo this, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/2977/files#r37436288>.
>